packages/babel-plugin-proposal-decorators/test/fixtures/2022-03-ordering/field/exec.js

```diff
@@ -0,0 +1,31 @@
+var log = [];
```

Comment:
This file is renamed from [decorators/exec.js](https://github.com/babel/babel/pull/15059/files#diff-218cf383abddc1b97731d8fc8d80db8b6696b56ca72efa4e56bd572fd0c38b15).